plover/dictionary/test_loading_manager.py

```diff
@@ -25,8 +25,6 @@ def __call__(self, filename):
         loader = MockLoader(files)
         with patch('plover.dictionary.loading_manager.load_dictionary', loader):
             manager = loading_manager.DictionaryLoadingManager()
-            manager.start_loading('a')
-            manager.start_loading('b')
```

Comment:
Ah ok. The start_loading function gets called later on in the normal way so I assumed this was redundant code. I'll re-instate.